.github/workflows/main.yml

```
@@ -12,93 +12,125 @@ env:
+        uses: actions/setup-node@v4
         with:
           node-version: ${{ env.NODE_VERSION }}
           cache: 'npm'
-      - run: npm ci
-      - run: npm run test:coverage
+          cache-dependency-path: ${{ matrix.app }}/package-lock.json
+      - name: Install ${{ matrix.app }} dependencies
+        run: npm ci
+      - name: Run ${{ matrix.app }} tests with coverage
+        run: npm run test:coverage
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)